shared/src/main/tut/README.md

```diff
@@ -709,6 +709,123 @@ The `UnitGroup` values provided with Squants are only samples and aren't intende
 We encourage users to make their own `UnitGroup` defintitions and submit them as PRs if they're generally
 applicable.
 
+## Formatters
+
+Squants provides an experimental API for formatting Quantities in the "best unit." For example, 
```

Comment:
I'm not sure. We've not had any feedback on the API from people who requested it. Once it's released, it's possible we'll want to change it based on feedback.

I'm fine moving unitgroups and formatters into an experimental package.